deploy/kubernetes/base/serviceaccount-csi-controller.yaml

```diff
@@ -6,7 +6,4 @@ metadata:
   name: ebs-csi-controller-sa
   labels:
     app.kubernetes.io/name: aws-ebs-csi-driver
-  #Enable if EKS IAM roles for service accounts (IRSA) is used. See https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html for details.
```

Comment:
Why are we removing this comment? Is it no longer relevant?

Reply:
They aren't critical to this change, so if you feel strongly I can put them back in, but I took the opportunity to remove all remaining `if eq .Release.Name "kustomize"` invocations in the chart as they were directly next to what I was already working on.

This particular comment is bad because:
1. It encourages manually editing the manifests, which is bad practice and we should not be encouraging (instead, users should use the Helm chart or Kustomize to make changes)
2. It creates a small additional burden on maintainers that they must understand the special properties of the `kustomize` release name
3. Explaining how IAM permissions interact with the driver is better left to `install.md`, where we document the various methods of granting IAM permissions, what policies the role needs, the existence of the managed policy, and other crucial information this short comment does not contain